hack/fork-packages.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+set -e
+set -o xtrace
+
+# fetch the packages if they don't exist
+for item in api apimachinery client-go
+do
+  if [ ! -d "forked/$item" ]; then
+    git clone https://github.com/kubernetes/$item forked/$item
```

Comment:
Do we need to checkout a specific version for the those packages?

Reply:
Not today

Reply:
As a follow up, would you pin each of these deps to a specific version, and document in the README.md which version they were pinned to?